HelpSource/Classes/Buffer.schelp

```diff
@@ -833,7 +833,8 @@ x.free; b.free;
 
 method:: query
 Sends a b_query message to the server, asking for a description of this buffer. The results are posted to the post window. Does not update instance vars.
-
+argument:: action
+An optional Function to be evaluated which will be passed the buffer's information as argument. Providing a function here will bypass this method's normal behaviour. A description of the buffer will not be posted.
 method:: updateInfo
```

Comment:
Suggestion, instead of " An optional Function to be evaluated which will be passed the buffer's information as argument." you could write:

An optional Function to be called with the query reply as an argument (code::["\b_query, bufnum, numFrames, numChannels, sampleRate]::).

Reply:
Ok, will do. The argument that is passed to the function is actually: [/b_info, bufnum, numFrames, numChannels, sampleRate]